lib/billomat/utils.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Billomat
+  # This class provides the possibility utility functions for data formatting
+  class Utils
+    def self.get_sufix(name)
```

Comment:
Since there's already `resource_name`, why not handle it within the model(s), e.g.:
```ruby
module Billomat
  module Models
    class Tax < Base
      def self.base_path
        '/taxes'
      end

      def self.base_name
        'taxes'
      end

      def self.resource_name
        'tax'
      end
    end
  end
end
```

`base_path` is so generic that it can probably be moved into `Billomat::Models::Base`:
```ruby
def self.base_path
  "/#{base_name}"
end
```

This way, you'd have:
```ruby
resp[@resource.base_name]['@total'].to_i

# or

resp[@resource.base_name][@resource.resource_name]
```

Reply:
I'm fine with this, too!